examples/bri-3/src/bri/identity/bpiAccounts/capabilities/getAllBpiAccounts/getAllBpiAccountQuery.handler.ts

```diff
@@ -0,0 +1,23 @@
+import { IQueryHandler, QueryHandler } from '@nestjs/cqrs';
+import { BpiAccountStorageAgent } from '../../agents/bpiAccountsStorage.agent';
+import { BpiAccountDto } from '../../api/dtos/response/bpiAccount.dto';
+import { GetAllBpiAccountsQuery } from './getAllBpiAccounts.query';
+
+@QueryHandler(GetAllBpiAccountsQuery)
+export class GetAllBpiAccountsQueryHandler
+  implements IQueryHandler<GetAllBpiAccountsQuery>
+{
+  constructor(private readonly storageAgent: BpiAccountStorageAgent) {}
+
+  async execute() {
+    const bpiSubjects = await this.storageAgent.getAllBpiAccounts();
+
+    return bpiSubjects.map((bp) => {
+      return {
```

Comment:
We can use current mapper here

Reply:
@Manik-Jain i think it would be better to leave todo until we decide which mapper to use? if we use automapper we will have to change this too

Reply:
Yeah, let's not duplicate work if the new mapper is coming.